blocksuite/framework/store/src/transformer/base.ts

```diff
@@ -34,6 +34,11 @@ export type SnapshotNode<Props extends object> = {
 export class BaseBlockTransformer<Props extends object = object> {
   protected _internal: InternalPrimitives = internalPrimitives;
 
+  /**
+   * The transformerConfigs will be set once the block transformer is created
+   */
+  transformerConfigs!: Map<string, unknown>;
```

Comment:
Should add the trasnformerConfig in constructor parameters to avoid this un-typesafe operation.